core/model/modx/processors/resource/unpublish.class.php

```diff
@@ -60,15 +71,6 @@ public function process() {
         return $this->success('',$this->resource->get(array('id')));
     }
 
-    /**
```

Comment:
Removing this public API function is a BC break. This cannot be merged in this current state until 3.x.

Reply:
The point of my review was to suggest the public API method `isSiteStart()` should not be removed at all. This change is not necessary and should not be done, even in 3.x without first releasing it with a deprecated mark. But again, I don't see removing this method to be necessary in the first place. Just don't use it for the purpose it was used prior to the other changes this PR makes. This SHOULD be integrated into 2.x for 2.7 release and can be if we can make that change.